fastboot-lt-2-10/instance-initializers/fastboot/head.js

```diff
@@ -15,5 +15,5 @@ export function initialize(instance) {
 
 export default {
   name: 'head-fastboot',
-  initialize: initialize
```

Comment:
The `initialize` function needs to be wrapped in a `if (typeof FastBoot != 'undefined') {...}` check to make sure `ember-cli-head` works in apps without `ember-cli-fastboot`.